sys/kern/src/arch/arm_m.rs

```diff
@@ -914,9 +889,9 @@ pub unsafe extern "C" fn SVCall() {
                 ",
                 exc_return = const EXC_RETURN_CONST as u32,
```

Comment:
Nit: indentation is a little weird now that the top `asm!("` has shifted right.  Sadly, it won't be autofixed by `rustfmt` because it's within the body of the `cfg_if!` macro.